csharp/Decode.cs

```diff
@@ -0,0 +1,14 @@
+//Script to decode Base64 text
+using System;
+
+public class Decode{
```

Comment:
```suggestion
class Decode{
```

Remove the unnecessary annotation from the class. We should remove them from the others, too.

Reply:
Make sure to get the other places we don't need `public class`, too.